README.md

```diff
@@ -140,38 +128,34 @@ _ = manager.rx.responseString(.get, stringURL)
 
 // URLHTTPResponse + Validation + String
 _ = manager.rx.request(.get, stringURL)
+    .validate(statusCode: 200 ..< 300)
+    .validate(contentType: ["text/json"])
     .flatMap {
-        $0
-            .validate(statusCode: 200 ..< 300)
-            .validate(contentType: ["text/json"])
-            .rx.string()
+        $0.rx.string()
```

Comment:
Since you made a helper for "responseJSON", maybe better to make a similar for responseString and get rid of this flatMap ?

Reply:
The helper is not mine. But it would be really nice to have a `responseString` helper. In fact, it would be really nice to have helpers for all response types, so that it feels as natural as the new validation methods. 😄